pkgs/applications/misc/mupdf/default.nix

```diff
@@ -64,11 +64,9 @@ stdenv.mkDerivation rec {
     substituteInPlace Makerules --replace "(shell pkg-config" "(shell $PKG_CONFIG"
   '';
 
-  # Use shared libraries to decrease size
-  buildFlags = [ "shared" ];
-
   makeFlags = [
     "prefix=$(out)"
+    "shared=yes"
```

Comment:
Maybe only when !isStatic?

Reply:
I'm not strictly opposed to making this change, but in practice there are other issues that would need to be addressed as well (the `.pc` file is wrong/broken for static libs since it doesn't include transitive dependencies), and I figure using `isStatic` might give the impression that this is a supported configuration when it's clearly not.

I'm not planning to address the further issues with static builds in this PR, I just happened to find this while helping someone on #dev:nixos.org earlier today.

Reply:
Let's not worry about that for now then